src/App.tsx

```diff
@@ -188,7 +188,7 @@ const App = (props: AppProps): JSX.Element => {
               to={`/?preferences=open&section=account&subsection=${params.get('tab') ?? 'account'}`}
             />
             <Redirect from="/app/:section?" to={{ pathname: '/:section?', search: `${queryParameters}` }} />
-            {pathName !== 'checkout-plan' && isMobile && isAuthenticated ? (
+            {pathName !== 'checkout-plan' && pathName !== 'checkout' && isMobile && isAuthenticated ? (
```

Comment:
For now, yes,
/checkout-plan takes the user to the Stripe checkout and /checkout is the pathname for the integrated checkout. For now you should keep both because the integrated checkout is only available for b2c payments and not b2b.